helm.go

```diff
@@ -38,5 +39,13 @@ func actionConfigInit(kubeInfo *KubeInformation) (*action.Configuration, error)
 		return nil, err
 	}
 
+	actionConfig.RegistryClient, err = registry.NewClient(
+		registry.ClientOptDebug(settings.Debug),
+		registry.ClientOptCredentialsFile(settings.RegistryConfig),
+	)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
what is this necessary for?

Reply:
the registry client is necessary to do oci interactions. Helm would complain if this is not initialized , but the documentation there is not that good:
`// registryClient provides a registry client` and `unable to lookup chart %q, missing registry client`